watson/cli.py

```diff
@@ -822,3 +822,111 @@ def sync(watson):
+        len(conflicting), width=dig))
+
+    if not no_dry_run:
+        return
+
+    if conflicting:
+        click.echo("Will resolve conflicts:")
+
+    date_format = 'YYYY-MM-DD HH:mm:ss'
+
+    for conflict_frame in conflicting:
+        original_frame = original_frames[conflict_frame.id]
+
+        # Print original frame
+        original_frame_data = {
+            'project': original_frame.project,
+            'start': original_frame.start.format(date_format),
+            'stop': original_frame.stop.format(date_format),
+            'tags': original_frame.tags
+        }
+        click.echo("frame {}:".format(style('short_id', original_frame.id)))
+        click.echo("{}".format('\n'.join('<' + line for line in json.dumps(
+            original_frame_data, indent=4, ensure_ascii=False).splitlines())))
+        click.echo("---")
+
+        # make a copy of the namedtuple
+        conflict_frame_copy = conflict_frame._replace()
+
+        # highlight conflicts
+        if conflict_frame.project != original_frame.project:
+            project = '**' + str(conflict_frame.project) + '**'
+            conflict_frame_copy = conflict_frame_copy._replace(project=project)
+
+        if conflict_frame.start != original_frame.start:
+            start = '**' + str(conflict_frame.start.format(date_format)) + '**'
+            conflict_frame_copy = conflict_frame_copy._replace(start=start)
+
+        if conflict_frame.stop != original_frame.stop:
+            stop = '**' + str(conflict_frame.stop.format(date_format)) + '**'
+            conflict_frame_copy = conflict_frame_copy._replace(stop=stop)
+
+        for idx, tag in enumerate(conflict_frame.tags):
+            if tag not in original_frame.tags:
+                conflict_frame_copy.tags[idx] = '**' + str(tag) + '**'
+
+        # Print conflicting frame
+        conflict_frame_data = {
+            'project': conflict_frame_copy.project,
+            'start': conflict_frame_copy.start.format(date_format),
+            'stop': conflict_frame_copy.stop.format(date_format),
+            'tags': conflict_frame_copy.tags
+        }
+        click.echo("{}".format('\n'.join('>' + line for line in json.dumps(
+            conflict_frame_data, indent=4, ensure_ascii=False).splitlines())))
+        resp = click.prompt(
+            "Select the frame you want to keep: left or right? (L/r)",
+            value_proc=options(['L', 'r']))
+
+        if resp == 'r':
+            # replace original frame with conflicting frame
+            original_frames[conflict_frame.id] = conflict_frame
+
+    # merge in any non-conflicting frames
+    for frame in merging:
+        start, stop, project, id, tags, updated_at = frame.dump()
+        original_frames.add(project, start, stop, tags=tags, id=id,
+                            updated_at=updated_at)
+
+    # overwrite the original frames file if any frames have changed
+    if original_frames is not None and original_frames.changed:
```

Comment:
Since watson.save() will use the default frames file, does the user really need to supply the path to the original frames file? Can we then just supply the path to the confict file and assume that the user wants to merge it into the default frames file?

We had originally based the design off of the discussion in issue #18 where the design was to specify both path to files. However, if we choose to use the default frames file, we may no longer need the first argument.